### PR TITLE
HackToberFest2020

### PR DESCRIPTION
**Pull request by sonalimahajan12**
# Title/Heading < Edit this part>

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change. < Edit this part>

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

-   [x] Bug fix (non-breaking change which fixes an issue)
-   [x] New feature (non-breaking change which adds functionality)
-   [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-   [x] This change requires a documentation update

# Checklist:

-   [x] My code follows the style guidelines of this project
-   [x] I have performed a self-review of my own code
-   [x] I have commented my code, particularly in hard-to-understand areas
-   [x] I have made corresponding changes to the documentation
-   [x] My changes generates no new warnings
-   [x] I have added tests that prove my fix is effective or that my feature works
-   [x] New and existing unit tests pass locally with my changes
-   [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by kaustubhgupta** [comment]
@sonalimahajan12 create the issue of the same and fill this PR template

**Comment by YashSaxena9** [comment]
Please follow the contribution guide :)

File: C++/Sliding Window Maximum (Maximum of all subarrays of size k).cpp [hunk 1]
```diff
@@ -0,0 +1,27 @@
+#include<bits/stdc++.h>
```
Comment (kaustubhgupta) [inline C++/Sliding Window Maximum (Maximum of all subarrays of size k).cpp:1]: Have a look at contribution guidelines

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]

**Comment by sonalimahajan12** [comment]
Hi kaustubh ,

can you open this issue again? I will do it again as per your requirements.

or should I again make pull request.

**Pull request closed without merging**